src/app/pages/BlockDetailPage/index.tsx

```diff
@@ -82,6 +92,17 @@ export const BlockDetailPage: FC = () => {
           </StyledDescriptionList>
         )}
       </SubPageCard>
+      <Card>
+        <CardHeader disableTypography component="h3" title={t('block.title')} />
+        <CardContent>
+          <Transactions
+            transactions={transactionsQuery.data?.data.transactions}
+            isLoading={transactionsQuery.isLoading}
+            limit={NUMBER_OF_ITEMS_ON_SEPARATE_PAGE}
+            pagination={true}
```

Comment:
I merged https://github.com/oasisprotocol/explorer/pull/79 in the meantime, so this needs to be migrated